submissions/amooChristianah/challenge3/solution3.js

```diff
@@ -0,0 +1,12 @@
+const getSmallest = (arr) => {
```

Comment:
You solved question 3 again here. This is an O(nlogn) solution. It is better than the one above. However, it fails the test case `nums = [1]`. The algorithm returns 1, and the expected answer is 2.